src/targets/kotlin/api.hbs

```diff
@@ -344,9 +344,14 @@ data class {{{name}}}(
         @SerializedName("{{{key}}}")
     {{/unless}}
     {{/if}}
-        val {{{name}}}: {{#if format}}@Format("{{{format}}}") {{/if}}{{{type}}}{{#unless @last}},{{/unless}}
```

Comment:
Kotlin's default indentation for a data class is meant to be 8 spaces so this was not a bug.